### PR TITLE
Add Control For Foundational Security Autoscaling 9

### PR DESCRIPTION
**Pull request by Priyanka-Chatterjee-2000**
### Checklist
- [ ] Issue(s) linked

### CODE REVIEW

File: query/autoscaling/autoscaling_group_uses_ec2_launch_template.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
+select
+  -- Required Columns
+  autoscaling_group_arn as resource,
+  case
+    when launch_template_id is not null then 'ok'
+    else 'alarm'
+  end as status,
+  case
+    when launch_template_id is not null then title || 'created from an EC2 launch template.'
```
Comment (misraved) [inline query/autoscaling/autoscaling_group_uses_ec2_launch_template.sql:9]:
```suggestion
    when launch_template_id is not null then title || ' created from an EC2 launch template.'
```

File: query/autoscaling/autoscaling_group_uses_ec2_launch_template.sql [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    else title || 'not created from an EC2 launch template.'
```
Comment (misraved) [inline query/autoscaling/autoscaling_group_uses_ec2_launch_template.sql:10]:
```suggestion
    else title || ' not created from an EC2 launch template.'
```

**Review by misraved — COMMENTED** [review]
Please take a look at the review comments. Thanks!!

**Review by rajlearner17 — APPROVED** [review]
LGTM after testing

**Review by misraved — APPROVED** [review]

**Pull request merged**